chart/cohort/templates/spark-history-server/deployment.yaml

```diff
@@ -49,6 +49,8 @@ spec:
           value: {{ .Values.sparkHistoryServer.s3.bucket }}
         - name: AWS_LOG_PATH
           value: {{ .Values.sparkHistoryServer.s3.eventsDir }}
+        - name: SPARK_HISTORY_OPTS
+          value: {{ .Values.sparkHistoryServer.logs.sparkHistoryOpts }}
```

Comment:
Pedantic:  I don't really see the need for the `logs` part of the identifier hierarchy.
I would be fine with just `sparkHistoryServer.sparkHistoryOps`.